cmd/root.go

```diff
@@ -25,3 +26,14 @@ func Execute() {
 		os.Exit(1)
 	}
 }
+
+func init () {
```

Comment:
Hi @sascha-andres, thanks for the PR.

It looks good, however I think opening database should be done in `main.go` instead here. The reasons are :

- When opening database, there is a possibility that panic happened. I personally think `init` must never panic, because `init` always executed, even if the package only imported and not used.
- For better package separation. As you can see, the [`Database`](https://github.com/RadhiFadlillah/shiori/blob/master/database/database.go#L10) is declared as interface because I have plan to support other DBMS in the future. The `cmd` is supposed to only use the `Database`, without caring what kind of database it is. That's why I think opening database should be done in `main.go`.